index.html

```diff
@@ -1,16 +1,78 @@
-<!DOCTYPE html>
-<html lang="en">
 
-<head>
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
+    <title>Cakes Project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
 <body>
-    <!-- Add your markup here -->
+  <div class="wrapper">
+  <!--header-->
+    <header class="header">
+      <img class="logo" src="/img/cake-logo-design-template-download_529200-598.webp">
+      <p>The best cakes in town delivered to your door.</p>
+    </header>
+    <nav class="nav-section">
+      <ul>
+        <li> <a class="nav-link active" href="#">HOME</a></li>
+        <li> <a class="nav-link active" href="#">CAKES</a></li>
+        <li> <a class="nav-link active" href="#">ORDERIN</a></li>
+        <li> <a class="nav-link active" href="#">LESSONS</a></li>
+        <li> <a class="nav-link active" href="#">ABOUT</a></li>
+      </ul>
+    </nav>
+  <!--end header-->
+ <!-- top container-->
```

Comment:
Could have done with a opening main tag here, but your sematic HTML is good otherwise :)